packages/drivers/odsp-driver-definitions/src/resolvedUrl.ts

```diff
@@ -145,13 +145,15 @@ export interface IOdspResolvedUrl extends IResolvedUrl, IOdspUrlParts {
 	isClpCompliantApp?: boolean;
 
 	/**
-	 * Context for given resolved URL
+	 * Context for given resolved URL. The context of a resolved URL is a string that contains
+	 * the resolved URL and the data store path of the resolved URL.
 	 */
 	context?: string;
 
 	/**
 	 * Name of the application that owns the URL. This hint is used by link handling logic which determines which
 	 * app to redirect to when user navigates directly to the link.
+	 * Can be ommited in case it is not necessary for the link handling logic.
```

Comment:
I'm not super familiar with this space, but I'm wondering if a user of this API will know when this is / is not true. If it's self-explanatory, then this is probably fine. But if not, we may want to expand on this a bit.